lib/rules/max-empty-lines/__tests__/index.js

```diff
@@ -129,3 +129,22 @@ testRule(rule, {
     column: 1,
   }],
 })
+
+testRule(rule, {
+  ruleName,
+  config: [ 2, { ignore: "comments" } ],
+
+  accept: [ {
+    code: "a{}\r\n\r\n/*\n\n\n\nhorse */\r\n\r\nb{}",
+  }, {
+    code: "a{}\r\n\r\n/** \r\n\r\n\r\n\nhor\r\n\r\n\r\nse */\r\n\r\nb{}",
```

Comment:
Be good added test between properties and value (`display: /* some long comments */ block;`, `display: block /* some long comments */;`, `display/* some long comments */: block;`).